test/ir/inference/test_trt_convert_activation.py

```diff
@@ -178,21 +178,25 @@ def generate_trt_nodes_num(attrs, dynamic_shape):
         # for static_shape
         clear_dynamic_shape()
         self.trt_param.precision = paddle_infer.PrecisionType.Float32
+        program_config.set_input_type(np.float32)
```

Comment:
For all files with the test_trt_convert test, we define the input type of the input data in the program configuration before creating the sample configuration.